vogue/server/__init__.py

```diff
@@ -45,8 +44,16 @@ def configure_app(app, config=None):
     app.client = client
     app.db = client[db_name]
     app.adapter = VougeAdapter(client, db_name=db_name)
-    app.register_blueprint(blueprint)
     app.register_blueprint(covid_blueprint)
+    app.register_blueprint(home_blueprint)
+    app.register_blueprint(common_trends_blueprint)
+    app.register_blueprint(prepps_blueprint)
+    app.register_blueprint(sequencing_blueprint)
+    app.register_blueprint(mip_blueprint)
+    app.register_blueprint(micro_blueprint)
+    app.register_blueprint(cancer_blueprint)
+    app.register_blueprint(genotype_blueprint)
+    app.register_blueprint(index_blueprint)
 
     if app.config['DEBUG'] == 1:
```

Comment:
```suggestion
    if app.config['DEBUG']:
```
Just a suggestion, if you want to change from `0,1` to `True, False` this would still work